cmd/abigen/namefilter_test.go

```diff
@@ -7,7 +7,6 @@
 // original code from which it is derived.
 //
 // Much love to the original authors for their work.
-// **********
```

Comment:
Why the removal? The geth files seem to have no copyright at all, so I'm not sure why we would modify this 🤔

Reply:
Never mind, this aligns with coreth 👍